frontends/web/src/routes/account/account.tsx

```diff
@@ -117,14 +121,26 @@ class Account extends Component<Props, State> {
         if (this.props.code !== prevProps.code) {
             this.onStatusChanged();
         }
-        if (this.props.deviceIDs.length !== prevProps.deviceIDs.length) {
+        if (this.state.deviceIDs.length !== this.deviceIDs().length) {
+            this.setState({ deviceIDs: this.deviceIDs() });
```

Comment:
again, why is this a state var? (state vars are for stuff that impacts rendering, `render()` is triggered). it is less code and less fragile to just call `this.deviceIDs()` where needed (or pass it to checkSDCards as an argument).

Reply:
fixed